src/middlewares/rateLimit.ts

```diff
@@ -1,17 +1,50 @@
 import VError from '@voiceflow/verror';
 import { NextFunction, Request, Response } from 'express';
+import { RateLimiterRes } from 'rate-limiter-flexible';
 
 import { AbstractMiddleware, RateLimitConfig } from '../types';
 
 // eslint-disable-next-line import/prefer-default-export
-export abstract class RateLimitMiddleware<S extends Record<string, any>, C extends RateLimitConfig> extends AbstractMiddleware<S, C> {
+export class RateLimitMiddleware<S extends Record<string, any>, C extends RateLimitConfig> extends AbstractMiddleware<S, C> {
   static throwAuthError(): never {
     throw new VError('Auth Key Required', VError.HTTP_STATUS.UNAUTHORIZED);
   }
 
-  // eslint-disable-next-line @typescript-eslint/ban-types
-  abstract verify(req: Request<{}>, _res: Response, next: NextFunction): Promise<void>;
+  static setHeaders(res: Response, rateLimiterRes: RateLimiterRes, maxPoints: number) {
+    res.setHeader('X-RateLimit-Limit', maxPoints);
+    res.setHeader('X-RateLimit-Remaining', rateLimiterRes.remainingPoints);
+    res.setHeader('X-RateLimit-Reset', new Date(Date.now() + rateLimiterRes.msBeforeNext).toString());
```

Comment:
```suggestion
    res.setHeader('X-RateLimit-Reset', new Date(Date.now() + rateLimiterRes.msBeforeNext).toISOString());
```
might be nicer for clients to parse this instead of however JS formats toString